dockerfiles/init/manifests/codenvy.env

```diff
@@ -86,17 +86,121 @@
 #CODENVY_SERVER_XMX=2048
 
 
-
 ########################################################################################
 #####                                                                              #####
-#####                                   DOCKER                                     #####
+#####                                 NETWORKING                                   #####
 #####                                                                              #####
-# IP Address
-#     The IP address of the Docker daemon that is running on your host. We do a 
-#     self-discovery to set this. You can combine this with values of DOCKER_HOST to 
-#     change communications from socket to TCP.
+#
+# Affects how the Codenvy server connects to the Docker Daemon, how the Che server and
+# workspaces establish connections to one another, and also how remote browser clients
+# should discover and connect to workspaces.
+#
+# Codenvy goes through a progression algorithm to establish the protocol, IP address and
+# port to establish communications with it is booting or starting a workspace.
+#
+# Browser --> Codenvy Server
+#    1. Default is 'http://localhost:${SERVER_PORT}/wsmaster/api'.
```

Comment:
Can you verify this?  If so, will remove.  I suspect that the networking evaluation cycle for Codenvy is quite different than che, and this is just che's cycle repeated for codenvy.